serialisation/src/test/java/org/ehrbase/serialisation/flatencoding/FlatConformanceTest.java

```diff
@@ -297,12 +296,6 @@ private void check(
 
     softAssertions.assertThat(unmarshal).isNotNull();
 
-    try {
-      new Validator(templateProvider.find(templateId).get()).check(unmarshal);
-    } catch (Exception e) {
-      softAssertions.fail(e.getMessage());
-    }
-
```

Comment:
Please add the new compostionValidate where ever you removed the old one.

Reply:
Tests added in `FlatConformanceValidationTest`